pytpcc/drivers/mongodbdriver.py

```diff
@@ -1140,6 +1140,10 @@ def get_server_status(self):
            del ss["transportSecurity"]
         if "metrics" in ss and "aggStageCounters" in ss["metrics"]:
            del ss["metrics"]["aggStageCounters"]
+        if "metrics" in ss and "operatorCounters" in ss["metrics"] and "expressions" in ss["metrics"]["operatorCounters"]:
+           del ss["metrics"]["operatorCounters"]["expressions"]
+        if "metrics" in ss and "operatorCounters" in ss["metrics"] and "match" in ss["metrics"]["operatorCounters"]:
+           del ss["metrics"]["operatorCounters"]["match"]
```

Comment:
Just a suggestion, but why not delete operatorCounters the way aggStageCounters is deleted above?
```suggestion
        if "metrics" in ss and "operatorCounters" in ss["metrics"]:
           del ss["metrics"]["operatorCounters"]
```

Reply:
Thank you very much for reviewing! My thought here was to delete the minimal amount of info to fix this issue for both SERVER-56422 and SERVER-56602 (assuming we have other operationCounters). Either way it looks like a temp solution as other $-prefixed metrics may be added in the future, so we probably should look into other ways to handle it  (I did created [HELP-25443](https://jira.mongodb.org/browse/HELP-25443) to get better understanding where we are with support of $-prefixed metrics in pymango).

Reply:
Makes sense, thanks for explaining!